alpha/lib/model/entry.php

```diff
@@ -3198,6 +3198,16 @@ public function setSearchProviderType($value)
 	{
 		$this->setSource($value);
 	}
+
+	public function setClonePendingEntries(array $clonePendingEntries)
```

Comment:
Instead if setClonePendingEntries, please use addClonePendingEntry and append it to the array internally.